### DISCUSSION TITLE
The ultimate note app quest

### DISCUSSION THREAD

**Discussion by giscus** [Jean-Tinland/giscus-comments.jeantinland.com#4] [Blog comments]
# The ultimate note app quest

This note app is a simple and efficient solution for taking random notes. With features like post-it style notes, drag-and-drop functionality, resizable notes, category assignment, markdown preview, and date association, this app provides a streamlined and intuitive note-taking experience.

https://www.jeantinland.com/blog/the-ultimate-note-app/

**Comment by seacowx** [comment]
Hi Jean, this is truly an amazing blog post and the resulting note app looks phenomenal. I'm wondering if there are plans to make the app public (e.g. publish the app on AppStore). Again, really impressive work!

**Reply by Jean-Tinland** [reply]
Hi @seacowx, first, thank you for your kind words, it means a lot!

As of now, I don't think the app can be published on the App Store as it is a web application built on web technologies. Notes are stored in a distant database, so no offline note-taking is possible. 

If I had to distribute the app, I think I'd take the open source, self-hosted route. That way users could have full control over their data, and I wouldn't need to host people's private notes.

Anyway, since at least one person other than me has taken an interest in the subject, I'll think about it, and maybe I'll come up with an idea!
